src/pass/pass.controller.ts

```diff
@@ -7,28 +7,38 @@ import { UpdatePassDto } from './dto/update-pass.dto';
 export class PassController {
   constructor(private readonly passService: PassService) {}
 
-  @Post()
-  create(@Body() createPassDto: CreatePassDto) {
-    return this.passService.create(createPassDto);
-  }
-
   @Get()
   findAll() {
     return this.passService.findAll();
   }
 
   @Get(':id')
   findOne(@Param('id') id: string) {
-    return this.passService.findOne(+id);
+    return this.passService.findByID(id);
+  }
+
+  @Get('user/:userId')
```

Comment:
liczba mnoga, trzeba byłoby to też zmienić w samym userze, chociaż mongoose dopisuje sobie 's' więc nawet jak zmienisz tutaj a w userze później to będzie działać,.
Ja tak obstaję  przy tej zmianie, po nawet patrząc na kod logicznej wydaje się szukać userId pośród users a nie user ;D

I co myślisz o tym, żeby userId zmienić na userID? Wyżej w findByID, akronim ID masz z dużych. Kiedyś gdzieś przeczytałam, że akronimy lepiej jest pisać z dużych liter, bo jest to bardziej czytelne i w sumie chyba racja ;D

Reply:
masz racje , na pewno będzie lepiej wyglądać

Reply:
zriobione